Packs/FeedUnit42v2/ReleaseNotes/1_0_50.md

```diff
@@ -0,0 +1,13 @@
+
+#### Integrations
+
+##### Unit 42 ATOMs Feed
+
+- Updated the Docker image to: *demisto/taxii2:1.0.0.90938*.
+- Added "rawJSON" key for the attack pattern indicator type.
+- Added "rawJSON" key for the report indicator type.
+- Added "rawJSON" key for the indicator type.
+- Added "rawJSON" key for the course of action indicator.
+- Added relationships creation for report indicator.
+- STIX report indicators pulled from the Unit 42 feed will now contain the *Report Object References* indicator field, which holds the IDs of related indicators.
```

Comment:
```suggestion
- Updated the Unit 42 feed so that STIX report indicators pulled from the feed will now contain the *Report Object References* indicator field, which holds the IDs of related indicators.
```